packages/edit-post/src/components/header/header-toolbar/style.scss

```diff
@@ -19,6 +19,21 @@
 			display: flex;
 		}
 	}
+	&.show-icon-labels {
+		button[aria-label] {
+			display: block;
```

Comment:
This line is causing all the buttons that are meant to be hidden on mobile to display:
<img width="320" alt="Screen Shot 2020-03-24 at 10 31 36 am" src="https://user-images.githubusercontent.com/8096000/77372845-c5e9cd00-6dba-11ea-9945-9542376350c6.png">

Reply:
Fixed that issue on mobile:

![image](https://user-images.githubusercontent.com/1477453/77596702-6d731680-6f05-11ea-934e-4684b153d8c7.png)

Reply:
And this is how that looks on desktop: 

![image](https://user-images.githubusercontent.com/1477453/77596803-b2974880-6f05-11ea-9525-f7614d0610e3.png)

aligned as @ZebulanStanphill suggested